gen/list.go

```diff
@@ -177,3 +181,40 @@ func (l *listGenerator) Equals(g Generator, spec *compile.ListSpec) (string, err
 
 	return name, wrapGenerateError(spec.ThriftName(), err)
 }
+
+func (l *listGenerator) zapMarshaler(
```

Comment:
Let's document what we're logging things as.

    // Slices are logged as JSON arrays.